components/legacy_sidebar/sidebar.test.jsx

```diff
@@ -216,6 +213,18 @@ describe('component/legacy_sidebar/sidebar_channel/SidebarChannel', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should match snapshot, when isDataPrefechEnabled is false', () => {
+        const wrapper = shallowWithIntl(
+            <Sidebar
+                {...{
+                    ...defaultProps,
+                    isDataPrefechEnabled: false,
+                }}
+            />,
+        );
+        expect(wrapper).toMatchSnapshot();
```

Comment:
What are you wanting to test here? Could we be more specific than just looking to see if we match a snapshot?

Reply:
test says `should match snapshot when isDataPrefechEnabled is disabled`

I am looking to not have `dataPrefetch` component mounted in the snapshot. it is a really simple change for prop and I felt it is good enough to have a snapshot.

Reply:
You know what you're looking for here, but others reviewing this in the future might not. I'm trying to avoid us running into cases where something breaks because someone changes something and just updates snapshots without understanding what actually changes since there's so much stuff in the snapshot.

Reply:
FYI i pretty much followed how we usually always write for test description including everything in the file for snapshots, Update the text based on your comment though now.